package.json

```diff
@@ -176,12 +177,14 @@
     "json5": "2.2.3",
     "plist": "3.0.5",
     "react": "18.2.0",
+    "react-beautiful-dnd": "^13.1.1",
     "react-dom": "18.2.0",
     "react-i18next": "12.2.0",
     "react-markdown": "8.0.5",
     "react-router-dom": "6.9.0",
     "recharts": "2.4.3",
     "sanitize-filename": "1.6.3",
+    "sanitize-html": "^2.11.0",
```

Comment:
GOG's changelogs are HTML, we sanitize any script or other crap (just in case) with this